src/Api/Model/Statistics/ArtistFollowingInfo.ts

```diff
@@ -0,0 +1,7 @@
+import { ItemsPerMonth } from './StatisticsResponse'
+
+export type ArtistFollowingInfo = {
+  readonly followingsPerMonth: ItemsPerMonth[]
```

Comment:
see suggestions below

```suggestion
  readonly followingsPerMonth: ItemsPerMonth
```